### PR TITLE
update tutorial link for dnn

### PR DESCRIPTION
**Pull request by LaurentBerger**
Insert dnn tutorial in http://docs.opencv.org/master/d9/df8/tutorial_root.html

### CODE REVIEW

File: doc/tutorials/dnn/table_of_content_dnn.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
+The dnn Functionality (dnn module) {#tutorial_table_of_content_dnn}
+=====================================
+
+Here you will learn the about the basic building blocks of the library. A must read and know for
```
Comment (catree) [inline doc/tutorials/dnn/table_of_content_dnn.markdown]: Probably you want to update also the description for these tutorials (here and below)?

Comment (LaurentBerger) [inline doc/tutorials/dnn/table_of_content_dnn.markdown]: Oops !
 I delete everything. Next commit

-   @subpage tutorial_dnn_googlenet

    *Compatibility:* \> OpenCV 3.2
    
    *Author:* 

    In this tutorial you will learn how to use opencv_dnn module for image classification by using GoogLeNet trained network from Caffe model zoo.

Do you know module author ?

Comment (alalek) [inline doc/tutorials/dnn/table_of_content_dnn.markdown]:
```
Compatibility: > OpenCV 3.3
```

for all tutorials.

```
Author: Vitaliy Lyudvichenko
```

for this tutorial.

```
Author: Dmitry Kurtaev
```

for Halide tutorials.

**Comment by alalek** [comment]
Thank you! :+1:

**Pull request merged**